File: my-folder/0392-is-subsequence/solution.cpp
class Solution {
public:
    bool isSubsequence(string s, string t) {
            int j=0, i=0;
            while(t[i]){
                if(s[j]!= t[i])
                i++;
                else {
                    i++;
                    j++;
                }
        }
         if(j==s.size() && i==t.size()) return true;
         else return false;

        
    }
};

File: my-folder/0392-is-subsequence/solution.py
class Solution:
    def isSubsequence(self, s: str, t: str) -> bool:
        n = len(s)
        i = 0
        for j in range(len(t)):
            if i < n and t[j] == s[i]:
                i += 1
    
        return n == i
